### PR TITLE
ux assurance for renewals

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#8724

*Description of changes:*
- changes based on ux assurance

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
looks good

File: ppr-ui/src/components/registration/RegistrationLengthTrust.vue [hunk 1]
```diff
@@ -1,7 +1,12 @@
 <template>
   <v-container fluid no-gutters class="white pa-0" v-if="summaryView">
     <v-card flat id="length-trust-summary">
-      <v-row no-gutters class="summary-header pa-2">
+      <v-row class="py-2 renewal-title" v-if="isRenewal">
```
Comment (kialj876) [inline ppr-ui/src/components/registration/RegistrationLengthTrust.vue]: do you need no-gutters here? Also, are you getting anything from nesting the `<h2>` within the `v-row / v-col`? Might be able to just have the `<h2>` with the `v-if` and the `class`

**Review by kialj876 — APPROVED** [review]
Looks good! One comment to consider, but it might not be needed. Also needs the test fixed!

File: ppr-ui/src/components/registration/RegistrationLengthTrust.vue (hunk 1 above)
Comment (jordiwes) [reply ppr-ui/src/components/registration/RegistrationLengthTrust.vue]: changed it to just the h2

**Pull request merged**